#goal definition
geometry_msgs/Pose goal_pose
---
#result definition
geometry_msgs/Pose result_pose
---
#feedback
nav_msgs/Path feedback_path


# DriveTo with obstacle avoidance, it invoces the move_base to create a plan and then follows the path

